scalableminds/webknossos
# Add frontend for zarr import

### URL of deployed dev instance (used for testing):
- [https://addfrontendforzarrimport.webknossos.xyz/](https://addfrontendforzarrimport.webknossos.xyz/)

### Steps to test:
- add and import zarr data:
    - without credentials
    [https://data-humerus.webknossos.org/data/zarr/scalable_minds/l4dense_motta_et_al_demo/segmentation](https://data-humerus.webknossos.org/data/zarr/scalable_minds/l4dense_motta_et_al_demo/segmentation)
    [https://data-humerus.webknossos.org/data/zarr/scalable_minds/l4dense_motta_et_al_demo/color](https://data-humerus.webknossos.org/data/zarr/scalable_minds/l4dense_motta_et_al_demo/color)
    - https:// with credentials (ask @fm3)
    - s3:// with credentials (ask @fm3)
- merge different data layers together

- labels in the credentials fields should change according to the protocol

Open:
- [x] add optionally displayable log
- [x] check that both datasources have same voxel size else warning when merging
- [x] check if dataset already exists
- [x] make layers unique without overriding old layers of same name

### Issues:
- fixes #6272 
- builds on #6335 

------
(Please delete unneeded items, merge only when none are left open)
